client/components/overviewWidget/AddToCart.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import axios from 'axios';
+import Sizes from './Sizes.jsx';
+import Quantities from './Quantities.jsx';
+
+class AddToCart extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      product: null,
+      styles: [],
+      sizes: [],
+      quantities: []
+    };
+  }
+
+  componentDidMount() {}
+
+  componentDidUpdate(prevProps) {
+    if (this.props.product !== prevProps.product) {
+      axios.get(`/api/fec2/hrnyc/products/${this.props.product.id}/styles`)
+        .then(({data}) => {
+          let skus = Object.values(data.results[0].skus);
+          let sizes = [];
+          let quantities = [];
+          skus.forEach((sku) => {
+            sizes.push(sku.size);
+            quantities.push(sku.quantity);
+          });
+          this.setState({
+            product: this.props.product,
```

Comment:
I believe setting your state from a prop is discourage by React
See: https://reactjs.org/docs/react-component.html#constructor 
There is a big yellow note at the bottom.